src/frontend/magnify/src/components/jitsi/MagnifyMeeting/index.tsx

```diff
@@ -35,6 +35,11 @@ export const MagnifyMeeting = ({ ...props }: MagnifyMeetingProps) => {
     apiRef.current?.addListener('participantLeft', handleParticipantsChanged);
     apiRef.current?.addListener('videoConferenceJoined', handleParticipantsChanged);
     apiRef.current?.addListener('videoConferenceLeft', handleParticipantsChanged);
+    apiRef.current?.addListener('participantRoleChanged', (event: any) => {
```

Comment:
I know that it isn't the case for other listeners yet, but what do you think of using constants for event names ? :)

Reply:
Agreed